bin/install.sh

```diff
@@ -40,7 +40,16 @@ make install_compilation_dependencies
 make install_runtime_dependencies
 
 # Install additional ROS packages
-sudo apt-get install ros-$ROS_DISTRO-vision-msgs ros-$ROS_DISTRO-geometry-msgs ros-$ROS_DISTRO-sensor-msgs ros-$ROS_DISTRO-audio-common-msgss
+if [[ ${ROS_DISTRO} == "noetic" || ${ROS_DISTRO} == "melodic" ]]; then
+  echo "Installing ROS dependencies"
+  sudo apt-get -y install ros-$ROS_DISTRO-vision-msgs ros-$ROS_DISTRO-geometry-msgs ros-$ROS_DISTRO-sensor-msgs ros-$ROS_DISTRO-audio-common-msgs ros-$ROS_DISTRO-usb-cam
+fi
+
+# Install additional ROS2 packages
+if [[ ${ROS_DISTRO} == "foxy" || ${ROS_DISTRO} == "humble" ]]; then
+  echo "Installing ROS2 dependencies"
+  sudo apt-get -y install ros-$ROS_DISTRO-usb-cam
```

Comment:
I think that we might need to add some more installation commands here as described in the `Installation` section in the description of #256?

Reply:
`vision-msgs` and `colcon` probably, and some additional similar to ROS1 above when more ROS2 nodes are added .

Reply:
yes, this PR targets develop, when merging it into `ros2` branch we can adapt it

Reply:
Oh alright, thanks!